client/src/main/java/org/apache/uniffle/client/util/ClientUtils.java

```diff
@@ -30,15 +30,15 @@ public class ClientUtils {
   // taskAttemptId is rest of 20 bit, max value is 2^20 - 1
   public static Long getBlockId(long partitionId, long taskAttemptId, long atomicInt) {
     if (atomicInt < 0 || atomicInt > Constants.MAX_SEQUENCE_NO) {
-      throw new RuntimeException("Can't support sequence[" + atomicInt
+      throw new IllegalArgumentException("Can't support sequence[" + atomicInt
```

Comment:
RssException may be better. Because we will collect the RssException to judge the failure apps caused by RSS.

Reply:
The root cause of this problem is illegal argument, perhaps `new RssException(msg, new IllegalArgumentException())`?

> RssException may be better. Because we will collect the RssException to judge the failure apps caused by RSS.

For this use case, I think warpping exception outside may be better, for example:

```java
try {
  run_rss();
} catch (e) {
  throw new RssException(msg, e);
}
```

Reply:
> The root cause of this problem is illegal argument, perhaps `new RssException(msg, new IllegalArgumentException())`?
> 
> > RssException may be better. Because we will collect the RssException to judge the failure apps caused by RSS.
> 
> For this use case, I think warpping exception outside may be better, for example:
> 
> ```java
> try {
>   run_rss();
> } catch (e) {
>   throw new RssException(msg, e);
> }
> ```

Make Sense.